test/marketplace/ZcToken.sol

```diff
@@ -17,6 +17,8 @@ contract ZcToken {
   string public name;
   string public symbol;
   uint8 public decimals;
+  
+  mapping (address => uint256) private balances;
```

Comment:
this is anti-pattern, lol. you forgot. 

the pattern is to use `balanceOfReturns` see the mock Erc20
https://github.com/Swivel-Finance/gost/blob/v2/test/mocks/Erc20.sol